definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+declare module '@reach/visually-hidden' {
+  declare export default React$StatelessFunctionalComponent<{
+    children: React$Node
+  }>;
```

Comment:
You must add `...` here, it's an inexact object, sorry I missed it earlier. 
Surprising the tests doesn't brake on it though.

Reply:
Yikes, nice catch! I'm using `v0.84` with objects not being exact by default, will fix this soon